docs/building-from-source.md

```diff
@@ -0,0 +1,55 @@
+# Building the SBOM tool from source.
+
+The SBOM tool is a cross-platform tool written in C# and is compiled using .NET Core 3.1. Follow the instructions below to build the tool from source.
+
+## Prerequisites
+* Please download and install [Dotnet SDK](https://dotnet.microsoft.com/en-us/download/dotnet/3.1) version 3.1.300 or greater.
```

Comment:
Maybe make link point to latest .NET SDK instead of 3.1?

Reply:
No, we require 3.1.* in our global.json, so they need this SDK